gpswearos/build.gradle

```diff
@@ -0,0 +1,68 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'com.android.gpswearos'
+    compileSdk 32
+
+    defaultConfig {
+        applicationId "com.android.gpswearos"
+        minSdk 30
```

Comment:
What is the lowest minSdk you can use with Wear? The app has a minSdk of 24, so ideally we support back to that, or whatever the minimum for Wear is.

Reply:
Looks like lowest minSdk for Wear 2.0 is 25, so let's use that - https://developer.android.com/training/wearables/upgrade#version

Reply:
> Looks like lowest minSdk for Wear 2.0 is 25, so let's use that - https://developer.android.com/training/wearables/upgrade#version

Ok, I will set the minSdk with 25.